ts/shielder-sdk-crypto/src/scalar.ts

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
the same comment as in #76 please flag somehow (e.g. by deprecation marker or in Jira) that there is some code duplication that needs to be removed as soon as possible (to avoid any divergence)